website/docs/getting-started/cluster-setup/kubernetes.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Kubernetes
+sidebar_position: 1
+---
+
+# Setup a Kubernetes cluster
+*Note that this guide is only helpful in setting up a development environment; this setup is not recommended for a production environment.*
+
+## Prerequisites
+* This guide assumes that you have a Kubernetes cluster setup, this could also be a [minikube](https://minikube.sigs.k8s.io/docs/start/) cluster.
```

Comment:
Because a minikube cluster **is** a Kubernetes cluster:

```suggestion
* This guide assumes that you have a Kubernetes cluster setup, this could for example be a [minikube](https://minikube.sigs.k8s.io/docs/start/) cluster.
```